frontendResourceBundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -57,6 +57,17 @@ action.GitMachete.BaseRebaseBranchOntoParentAction.text=_Rebase Branch onto Pare
 action.GitMachete.BaseResetBranchToRemoteAction.description.enabled={0} branch ''{1}'' to its remote tracking branch
 action.GitMachete.BaseResetBranchToRemoteAction.action-name=Re_set to Remote
 action.GitMachete.BaseResetBranchToRemoteAction.description-action-name=Reset to remote
+action.GitMachete.BaseResetBranchToRemoteAction.info-dialog.dont-show-again=Don't Show Again
```

Comment:
Hmm why single single quote here but double single quote in other cases? :thinking:

Reply:
The pair of single-quotes escape format placeholders (like `{0}`). Additionally, double single quote reduces to a single single-quote. Single signle-quote is just a single single-quote. :rofl: 

Eventually, it looks correctly (see the image above).

Reply:
Aaaah okay, so the only potential error would be to first declare prop as `hello '{0}'` and then try to pass argument to this escaped '{0}', right? This should be anyway captured by I18nFormatChecker (you can verify at your convenience btw)

Reply:
verified